### DISCUSSION TITLE
Preferred way to update appimage version?

### DISCUSSION THREAD

**Discussion by tomhusband** [Alex313031/thorium#374] [Q&A]
I'm on a Linux Mint laptop and installed the appimage version. Is there a preferred way to be notified of and install an update?

Many thanks, I'm really liking this browser.

**Comment by froeb** [comment]
The scenario you're describing is indeed common in the Linux community. Many software projects update their software on GitHub but do not provide their packages in standard Linux repositories. This can be challenging for users who prefer the convenience of package managers for installation and updates.

Automatic Download and Installation from GitHub:

    Existing Tools: There are existing tools and scripts that can automate the process of downloading and installing software from GitHub. These tools typically work by fetching the latest release from a GitHub repository and then either building from source or installing pre-built binaries. For example, tools like git-clone and wget can be used in scripts to automate these tasks.

    Custom Scripting: Writing a script to automate this process is feasible. Such a script would:
        Check for the latest release of the software on GitHub.
        Download the release (source code or binaries).
        Optionally compile the source code if binaries are not available.
        Install the software on the system.

    Challenges:
        Dependency Management: One of the main challenges is managing dependencies for each software package, which can vary widely.
        Variability in Build Processes: Different software might have different build processes, which can complicate the automation.
        Security Concerns: Automatically downloading and installing software can pose security risks, especially if not properly vetted.
        
 I would be interested in anyone coming up with either a general solution or with a solution especially designed for Thorium.

**Comment by froeb** [comment]
Try this: https://github.com/froeb/manage-thorium

**Reply by tomhusband** [reply]
Thanks, I'll take a look.
